doc/classes/PhysicsBody2D.xml

```diff
@@ -33,6 +33,12 @@
 			</argument>
 			<description>
 				Returns an individual bit on the collision mask.
+
+				By default, the first bit is set:
+				[codeblock]
+				print(get_collision_layer_bit(0)) # True
+				[/codeblock]
+				This ensures that newly created physics bodies collide by default.
```

Comment:
I don't think we need this information duplicated here. It's probably better to only have the behavior of the collision layer/mask documented in the collision layer/mask sections below.

Reply:
> I don't think we need this information duplicated here.

Based on what, though? The idea is to make sure people don't miss it. Good documentation should duplicate important information. Of course it's easier when you have a doc tool that can include stuff.

Reply:
Also... too much information is the _least_ of the documentation's concerns right now.